lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -277,21 +277,18 @@ sub engine_workit {
                 }
 
                 # treat "no sync necessary" as success as well
-                my $exit = $status->result // 0;
+                my $result = $status->result // 'exit code 0';
```

Comment:
This looks very unusual. I must admit I am not yet convinced by turning everything into a string here. Is there no alternative?

Reply:
Then you misunderstand. I'm not turning everything into a string, everything already was a string but we pretended it wasn't. That's the problem.

Reply:
And there are many alternatives. I could use complex data structures for success... but that only increases the number of special cases and things to go wrong without really adding value (aside from looking fancy). What you see in this patch is simply the most defensive solution i could think of.